overlays/DIMO/dimo.vspec

```diff
@@ -66,3 +66,73 @@ Vehicle.Powertrain.TractionBattery.Charging.AddedEnergy:
   datatype: float
   unit: kWh
   description: Amount of charge added to the high voltage battery during the current charging session, expressed in kilowatt-hours.
+
+Vehicle.ADAS.CruiseControl.CruiseFollowTime:
+  datatype: float
+  type: actuator
+  description: Tesla cruise follow distance seems to be directly proportional to time, but is not seconds
+
+Vehicle.MaxSpeedAllowed:
+  datatype: float
+  type: actuator
+  unit: km/h
+  description: Max speed the vehicle can achieve while Speed Limit Mode is active
+
+Vehicle.ADAS.ObstacleDetection.Forward.Setting:
+  datatype: string
+  allowed: ['AVERAGE', 'EARLY', 'LATE', 'DEFAULT', 'OFF']
+  type: actuator
+  description: Forward collision warning settings, possible values are 'Average', 'Early', 'Late', 'Default', and 'Off'
+
+Vehicle.GuestModeEnabled:
+  datatype: boolean
+  type: actuator
+  description: Whether Guest Mode is enabled in the vehicle, true if Guest Mode is active, false otherwise
+
+Vehicle.Powertrain.TractionBattery.IdealRange:
+  datatype: float
+  type: sensor
+  unit: km
+  description: Remaining ideal range in meters using only battery.
+
+Vehicle.ADAS.LaneDepartureDetection.Level:
+  datatype: string
+  allowed: ['ASSIST', 'NONE', 'WARNING', 'DEFAULT']
+  type: actuator
+  description: The level of Lane Departure Detection, possible values are 'Assist', 'None', 'Warning', and 'Default'
+
+Vehicle.VehicleIdentification.Trim:
```

Comment:
I feel like this could do the job? 

https://github.com/DIMO-Network/VSS/blob/b4eb846eaf29e1fc94799652ecd96d6d066f9810/spec/Vehicle/Vehicle.vspec#L132